packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -590,8 +588,8 @@ export class ApexDebug extends LoggingDebugSession {
     this.sendResponse(response);
   }
 
-  private getSessionIdleTimer(): NodeJS.Timer[] {
-    const timers: NodeJS.Timer[] = [];
+  private getSessionIdleTimer(): Array<ReturnType<typeof setTimeout>> {
+    const timers: Array<ReturnType<typeof setTimeout>> = [];
```

Comment:
@gbockus-sf the syntax of `Array<ReturnType<typeof setTimeout>>` is more verbose, and (IMO) isn't as easy to grok as `NodeJS.Timer[]`.  I changed the type here back to `NodeJS.Timer[]` and that worked (for getSessionIdleTimer(), at least).  If not needed, my preference would be to  go back to NodeJS.Timer[].  I suspect thought there might have been a reason.  If so, what was it?

Reply:
I was originally running into a compile issue (or maybe it was when having ts-jest do the compilation although using the same tsconfig so I dunno why it was different) I just tired changing it back and it compiled and the tests passed without issue locally.  I'll see if it was specific to the CI build or something 🤷 I grabbed the change from https://stackoverflow.com/questions/51040703/what-return-type-should-be-used-for-settimeout-in-typescript but honestly don't remember the specifics of how/why it was causing an issue.

Reply:
Ah the dreaded only fails on the CI build unit test 
```
FAIL test/unit/adapter/apexDebugVariablesHandling.test.ts
@salesforce/salesforcedx-apex-debugger:   ● Test suite failed to run
@salesforce/salesforcedx-apex-debugger:     src/adapter/apexDebug.ts:594:7 - error TS2345: Argument of type 'number' is not assignable to parameter of type 'Timer'.
@salesforce/salesforcedx-apex-debugger:     594       setTimeout(() => {
@salesforce/salesforcedx-apex-debugger:               ~~~~~~~~~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     595         this.warnToDebugConsole(
@salesforce/salesforcedx-apex-debugger:         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     ... 
@salesforce/salesforcedx-apex-debugger:     601         );
@salesforce/salesforcedx-apex-debugger:         ~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     602       }, DEFAULT_IDLE_WARN1_MS),
@salesforce/salesforcedx-apex-debugger:         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger: FAIL test/unit/adapter/apexDebug.test.ts
@salesforce/salesforcedx-apex-debugger:   ● Test suite failed to run
@salesforce/salesforcedx-apex-debugger:     src/adapter/apexDebug.ts:594:7 - error TS2345: Argument of type 'number' is not assignable to parameter of type 'Timer'.
@salesforce/salesforcedx-apex-debugger:     594       setTimeout(() => {
@salesforce/salesforcedx-apex-debugger:               ~~~~~~~~~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     595         this.warnToDebugConsole(
@salesforce/salesforcedx-apex-debugger:         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     ... 
@salesforce/salesforcedx-apex-debugger:     601         );
@salesforce/salesforcedx-apex-debugger:         ~~~~~~~~~~
@salesforce/salesforcedx-apex-debugger:     602       }, DEFAULT_IDLE_WARN1_MS),
```
The updated type was the fix for these tests I found.  Would a code comment about why that type is used be useful?